app/model/commands/ActiveAssetCommand.scala

```diff
@@ -9,51 +9,44 @@ import data.DataStores
 import model.commands.CommandExceptions._
 import util._
 
-case class ActiveAssetCommand(atomId: String, params: ActivateAssetRequest, stores: DataStores,
-                              youTube: YouTube, user: PandaUser)
-  extends Command
-    with MediaAtomImplicits
-    with Logging {
+case class ActiveAssetCommand(
+  atomId: String,
+  activateAssetRequest: ActivateAssetRequest,
+  stores: DataStores,
+  youtube: YouTube,
+  user: PandaUser
+) extends Command with MediaAtomImplicits with Logging {
 
   type T = MediaAtom
 
   def process(): T = {
-    log.info(s"Request to $params in $atomId")
+    log.info(s"Request to set active asset atom=$atomId version=${activateAssetRequest.version}")
+
+    if (atomId != activateAssetRequest.atomId) {
+      AtomIdConflict
```

Comment:
Should we be using `AtomIdConflict` here since ending up here would be a symptom of of something else having gone wrong?

Reply:
I was thinking we should add this check to all the endpoints, otherwise we're trusting the id in the url path matches the id in the request body. We shouldn't reach here, and if we do its probably safer to fail early. What do you think?

Reply:
Makes sense I guess. What do you think could cause for the two not to match? It might still save confusion to reserve the different error name for this case, sth like `nonMatchingAtomIds` maybe because `AtomIdConflict` is used when creating atoms if the new random id generated is already in use.